packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -81,10 +81,19 @@ function getNackReconnectInfo(nackContent: INackContent) {
  * Implementation of IDocumentDeltaConnection that does not support submitting
  * or receiving ops. Used in storage-only mode.
  */
+const clientNoDeltaStream: IClient = {
+    mode: "read",
+    details: { capabilities: { interactive: true } },
+    permission: [],
+    user: { id: "unknown" }, // we need some "fake" ID here.
```

Comment:
@anthony-murphy - curious in your feedback. I want to deliver much simpler programming model, where "connected" means "self" in in audience. But NoDeltaStream breaks that promise. I can do either hack like you see here or exclude specifically NoDeltaStream in connection handler and not wait for join signal. The problem with later - it breaks that simple promise and pushes complexity to consumers. So, I'd prefer the simpler solution (what you see here), but that requires generation of fake user :(

Thoughts?

Reply:
I don't mind the fake value, although I haven't scanned through the code to see how user id is used.

Reply:
I would put "storage-only" or "No Delta Stream" or something in the value though, I guess.